src/device/index.ts

```diff
@@ -71,14 +71,30 @@ const device = (ctx: Context, prefix: string | undefined) => {
   }
 
   // get device node information including the assigned stake and, if running, the Docker container
-  const node = async () => {
-    const address = await ctx.wallet().address()
-    const deviceWallet = await (await volume()).read()
+  const node = async (remoteStake?: string) => {
+    const stake = await (async () => {
+      if (remoteStake === undefined) {
```

Comment:
This is a negative equivalence check where I think it would be clearer to use a positive one, i.e. `if (remoteStake) {...`